mmv1/third_party/terraform/services/datafusion/resource_data_fusion_instance_test.go

```diff
@@ -207,3 +213,40 @@ resource "google_data_fusion_instance" "basic_instance" {
 }
 `, context)
 }
+
+func TestAccDatafusionInstance_DeletionProtection(t *testing.T) {
+	t.Parallel()
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckDataFusionInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatafusionInstanceDeletionProtection("us-central1"),
+			},
+			{
+				ResourceName:            "google_data_fusion_instance.instance",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"labels", "terraform_labels", "deletion_protection"},
+			},
```

Comment:
Can you try running this test locally? Mostly the post test destroy test should fail. 
To fix that we need to add another step that sets deletion_protection = false after the test.
smth like: {Config: testAccDatafusionInstanceDeletionProtectionFalse("us-central1")}
where
func testAccDatafusionInstanceDeletionProtectionFalse(region string) string {

	return fmt.Sprintf(`
	resource "google_data_fusion_instance" "instance" {
        name   = "my-instance"
        region = "%s"
        type   = "BASIC"
	deletion_protection = false
}
`, region)
}